src/cursor.ts

```diff
@@ -3,20 +3,25 @@ import {
   ReferenceExpression,
   SelectQueryBuilder,
   StringReference,
+  sql,
 } from "kysely";
 
+type NullOrder = "NULLS LAST" | "NULLS FIRST";
```

Comment:
Probably makes sense to type this to something a bit more db-agnostic. Just `LAST | FIRST` maybe?